packages/design-system/src/Collapsible/Collapsible.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Collapsible } from "./Collapsible";
+import { CollapsibleProps } from "./Collapsible.types";
+import { Text } from "../Text";
+
+export default {
+  title: "Design System/Collapsible",
+  component: Collapsible,
+  parameters: {
+    docs: {
+      description: {
+        component:
+          "Collapsibles can expand when clicked on. They allow you to hide content that is not immediately relevant to the user.",
+      },
+    },
+  },
+} as ComponentMeta<typeof Collapsible>;
+
+// eslint-disable-next-line react/function-component-definition
+const Template: ComponentStory<typeof Collapsible> = (args) => {
+  return <Collapsible {...args} />;
+};
+
+export const CollapsibleStory = Template.bind({});
+CollapsibleStory.storyName = "Collapsible";
+CollapsibleStory.args = {
+  title: "This is a collapsible",
+  isOpen: true,
+  onHeaderClick: (isOpen: CollapsibleProps["isOpen"]) => {
+    alert(`Collapsible ${isOpen ? "expanded" : "collapsed"}!`);
```

Comment:
I saw similar behavior on the [button compoenent's onClick](https://design-system.appsmith.com/?path=/story/design-system-button--button-story). Any suggestions on what we can do instead?

Reply:
That alert exists because there's no other visual cue that the button has been clicked in the story. In this story, our visual cue is the content of the collapse hiding and showing itself, so an alert on top of this just feels extra.